lumigator/backend/backend/api/routes/experiments.py

```diff
@@ -67,9 +65,9 @@ def get_experiment_result(
 def get_experiment_result_download(
     service: JobServiceDep,
     experiment_id: UUID,
-) -> ExperimentResultDownloadResponse:
+) -> JobResultDownloadResponse:
     """Return experiment results file URL for downloading."""
-    return ExperimentResultDownloadResponse.model_validate(
+    return JobResultDownloadResponse.model_validate(
```

Comment:
the create_job function returns JobResultDownloadResponse and we were saying it was ExperimentResultDownloadResponse. This worked before because those classes were identical. But now they aren't (because experiment_id is now a str and not a uuid)